lib/git-shell-out-strategy.js

```diff
@@ -626,7 +626,8 @@ export default class GitShellOutStrategy {
   }
 
   async getRemotes() {
-    let output = await this.getConfig(['--get-regexp', '^remote..*.url$'], {local: true});
+    // eslint-disable-next-line no-useless-escape
+    let output = await this.getConfig(['--get-regexp', '^remote\..*\.url$'], {local: true});
```

Comment:
I think you need to double-escape the `\` (hence the linter complaining about it):

```
> console.log('foo\.bar')
foo.bar
undefined
> console.log('foo\\.bar')
foo\.bar
undefined
```